apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/RequestHeaders.java

```diff
@@ -190,6 +207,21 @@ public static Boolean returnIncludeAddressFieldsValue(String headerValue) {
         : Boolean.TRUE;
   }
 
+  /**
+   * Return a TRUE / FALSE from VALID_HEADER_VALUES_INCLUDE_IDENTIFIERS header
+   *
+   * @param headerValue a String containing Boolean value in string form
+   * @return True or False.
+   */
+  public static Boolean returnIncludeTaxValue(String headerValue) {
+    return (headerValue == null
+            || headerValue == ""
+            || headerValue.equalsIgnoreCase("FALSE")
+            || !headerValue.equalsIgnoreCase("TRUE"))
+        ? Boolean.FALSE
+        : Boolean.TRUE;
+  }
+
```

Comment:
Couldn't we simplify this as:

return (headerValue != null && headerValue.equalsIgnoreCase("TRUE"));